src/components/AllBooks/index.js

```diff
@@ -1,9 +1,7 @@
 import { Link } from "react-router-dom";
-import { Container } from "./allBooks.styles";
-import { Card } from "./allBooks.styles";
-import { Genre } from "./allBooks.styles";
+import { Container, Card, Genre } from "./allBooks.styles";
 
-export const AllBooks = (props) => {
+const AllBooks = (props) => {
```

Comment:
I'm not sure but I dont think you need "export default" on dumbcomponents(without state)